Annotato/Annotato/Persistence/PersistenceService/OnlinePersistenceService/OnlinePersistenceService+AnnotationsPersistence.swift

```diff
@@ -13,4 +13,13 @@ extension OnlinePersistenceService: AnnotationsPersistence {
     func deleteAnnotation(annotation: Annotation) async -> Annotation? {
         await remotePersistence.annotations.deleteAnnotation(annotation: annotation)
     }
+
+    func createOrUpdateAnnotationForLocal(annotation: Annotation) -> Annotation? {
```

Comment:
- Can you declare the function in the separate `DocumentsPersistence.swift` and `AnnotationPersistence.swift` protocols:

For example:
<img width="669" alt="Screenshot 2022-04-05 at 11 46 30 PM" src="https://user-images.githubusercontent.com/71375383/161793577-98932e8a-c7d0-42fc-8512-c56f2767be05.png">

- Also can you name the function as `createOrUpdateDocuments(documents: [Document])`
Same for annotations. Instead of naming it as `fastForward...`. `fastForward` lacks clarity.

- Also make createOrUpdateDocument (the singular one) private.
You only need to expose createOrUpdateDocuments (the plural one).

- Then, you implement them where needed. You can choose not to implement them in areas that won't be used for now and just throw a fatal area.